tables/de-g2-detailed.ctb

```diff
@@ -0,0 +1,60 @@
+# liblouis: German Grade 2 Braille (with capitals)
+#
+# Das System der deutschen Blindenschrift (2015/2018)
+# http://www.bskdl.org/textschrift.html
+#
+# The idea of this table is to be used with screen readers and
+# note-takers for both forward and backward translation. Forward
+# translation in this table is designed to give as much information as
+# possible in order for the result to be back-translatable.
```

Comment:
I think these two sentences are misleading (also in the g0 and g1 tables). And also that the g0 and g1 tables have `#+direction: both` in the detailed versions and `#+direction: forward` in the normal versions. The detailed and normal versions don't differ that much, not enough to say that one is back-translatable and the other isn't. I would say they are both back-translatable. One just provides a more detailed braille representation so back-translation is more exact.

Dunno, maybe I am missing something, maybe the differences between the variants are bigger than I think. https://github.com/liblouis/liblouis/issues/1175 will shed more light on things for me.

Reply:
No, I don't think you are missing anything. It is exactly as you say. One just provides a more detailed braille representation so back-translation is more exact.

I tried to improve these two sentences.

I agree with regards to the `#+direction:` meta data. Do you want to change this within this PR?

Reply:
Thanks. The description of de-g2-detailed.ctb is good now. However the text in de-g2.ctb is still not quite right IMO. The first part is correct:

    # This table is primarily meant to be used for forward translation.

But the second part is still misleading. The way it is written now suggests that the fact that the generated braille is not ideal for back-translation, because it is not detailed enough, is the reason why the table is only for forward translation, and that de-g2-detailed.ctb fixes this problem:

    # The generated Braille is really meant for embossing and is not ideal
    # for back-translation. Please, use the detailed tables in contexts
    # where back-translation is an issue.

But the real reason why the table is only for forward translation is because the table is not good enough and that German Kurzschrift is simply not very suited for back-translation with a computer. In other words what you wrote in de-g2-detailed.ctb ("due to the lack of consideration for back-translation ...") equally applies here.

> I agree with regards to the `#+direction`: meta data. Do you want to change this within this PR?

No, not necessarily in this PR. But I think it would be a good idea to do it now so we don't forget.

Reply:
OK, I improved the description in `de-g2.ctb` and marked it for both directions

Reply:
Thanks. Description looks good now.

But why mark it for both directions? It really isn't. (Have you ever tried to back-translate with this table?) And didn't we agree that there is not a big difference between the tables w.r.t. back-translation? But now de-g2-detailed.ctb is marked `forward` and de-g2.ctb is marked `both`.

Reply:
OK, I dropped the commit that changed it to `#+direction: both`